ecosystem/node-checker/src/evaluators/metrics/network/peers.rs

```diff
@@ -0,0 +1,357 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+/// These evaluators are only valuable in certain contexts. For example, this is
+/// not a useful evaluator for node registration for the AITs, since each node
+/// is running in their own isolated network, where no consensus is occurring.
+/// This is useful for the AIT itself though, where the nodes are participating
+/// in a real network.
+
+///
+use super::{
+    super::{
+        common::{get_metric, GetMetricResult},
+        types::{MetricsEvaluatorError, MetricsEvaluatorInput},
+    },
+    CATEGORY,
+};
+use crate::{
+    configuration::EvaluatorArgs,
+    evaluator::{EvaluationResult, Evaluator},
+    evaluators::{metrics::common::Label, EvaluatorType},
+};
+use anyhow::Result;
+use clap::Parser;
+use once_cell::sync::Lazy;
+use poem_openapi::Object as PoemObject;
+use prometheus_parse::Scrape as PrometheusScrape;
+use serde::{Deserialize, Serialize};
+
+////////////////////////////////////////////////&//////////////////////////////
```

Comment:
nit: why `&`? 😄

Reply:
at this point it's just an easter egg for you @JoshLind heheheh